scripts/install/functions.sh

```diff
@@ -80,48 +80,49 @@ function update_projects_versions {
 }
 
 function create_shell_app {
-    ( cd ${INSTALLATION_DIR} && ng new ${1} --style=scss --routing=false)
+    ( cd ${INSTALLATION_DIR} && ng new ${1} --style scss --no-routing)
 }
 
 function add_b2b {
     if [ "${ADD_B2B_LIBS}" = true ] ; then
-        ng add --skip-confirmation @spartacus/organization@${SPARTACUS_VERSION} --interactive false
-        ng add --skip-confirmation @spartacus/checkout@${SPARTACUS_VERSION} --interactive false --features="Checkout-B2B" --features="Checkout-Scheduled-Replenishment"
+        ng add @spartacus/organization@${SPARTACUS_VERSION} --skip-confirmation --no-interactive
```

Comment:
Some ng add command modules were removed this suffix `@${SPARTACUS_VERSION}` and some others no.  Is that intentional?